app/scripts/modules/tencentcloud/src/pipeline/stages/disableCluster/DisableCluster.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { AccountService, IStageConfigProps, NgReact, StageConfigField } from '@spinnaker/core';
+import { get } from 'lodash';
+const { AccountRegionClusterSelector } = NgReact;
+
+interface IProps extends IStageConfigProps {}
+interface IState {
+  accounts: any[];
```

Comment:
Can we avoid using `any` here in favor of a strongly-typed tencentcloud account? (Same comment for all other any-typed accounts in other files.)